desktop/cmp/form/FormField.js

```diff
@@ -205,7 +207,10 @@ FormField.propTypes = {
      * Optional function for use in readonly mode. Called with the Field's current value
      * and should return an element suitable for presentation to the end-user.
      */
-    readonlyRenderer: PT.func
+    readonlyRenderer: PT.func,
+
+    /** The indicator to display next to a required field. Defaults to span(' *'). */
```

Comment:
+ Think we should just say that it defaults to '*' -- here and in the formField.  Don't need to give the technical details of how we accomplished the spacing, etc.. 

+  This should be of type PT.element here.

Reply:
>This should be of type PT.element here.

I agree, but that breaks. If you give it an element in fieldDefaults, the page will fail to load.